### ISSUE TITLE
Jetpack.com: decorative gridicons are announced as images

### ISSUE THREAD

**Issue by jkguidaven** [Automattic/jetpack#29801]
## What's the issue
On the Jetpack.com homepage, many gridicons are announced as images by screen readers while they should be ignored.

## How to reproduce the issue
- Visit https://jetpack.com/
- Open a screen reader of your choice
- Notice that many gridicons in the page are decorative, but are announced as images (with no alternate text) by screen readers

## Expected behavior
Screen readers should ignore decorative icons.

## Captures

_These decorative checkmarks are gridicons_
<img width="400" src="https://user-images.githubusercontent.com/1620183/228360771-071cd14e-d311-4d54-80df-8dcf3833512b.png" alt=""/>

_This decorative start is a gridicon_
<img width="400" src="https://user-images.githubusercontent.com/1620183/228360865-2587e748-f350-4209-bcd6-cbcf05d9cb24.png" alt=""/>

**Comment by jkguidaven** [comment]
diff: D106529-code
